sdks/python/apache_beam/io/gcp/bigquery_file_loads.py

```diff
@@ -1101,6 +1101,18 @@ def _load_data(
          of the load jobs would fail but not other. If any of them fails, then
          copy jobs are not triggered.
     """
+    self.reshuffle_before_load = not util.is_compat_version_prior_to(
```

Comment:
Structural comments:

 - This reshuffle should be added outside of this transform. Make it the responsibility of the caller to ensure stable inputs.
 - It is be nicer to fully fork `expand` whenever we move to a new update-incompatible version. Basically freeze the old one and leave it behind (like https://github.com/apache/beam/blob/32620186a0029106b1396560257d68663891021d/sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/Reshuffle.java#L87). You still get bugfixes applied to helper transforms, but keep `expand` itself straight-line code except for the top-level decision of which version to expand.